paasta_tools/paasta_cli/utils.py

```diff
@@ -436,12 +436,17 @@ def run_paasta_serviceinit(subcommand, master, service, instancename, cluster, *
         app_id_flag = "--appid %s " % kwargs['app_id']
     else:
         app_id_flag = ''
-    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s%s %s' % (
+    if 'delta' in kwargs and kwargs['delta']:
+        delta = "--delta %s" % kwargs['delta']
+    else:
+        delta = ''
+    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s%s %s %s' % (
```

Comment:
It is true, but kinda an artifact of paasta_servceinit being the entrypoint for every type of emergency remote command. I think that is ok.